### PR TITLE
QNC serialization fix

### PR DESCRIPTION
**Pull request by Przemog1**

### CODE REVIEW

**Comment by Przemog1** [comment]
almost done

File: include/irr/asset/CQuantNormalCache.h [hunk 1]
```diff
@@ -363,6 +340,76 @@ class CQuantNormalCache
...
+			bufferPtr = static_cast<uint8_t*>(buffer.buffer.get()->getPointer());
+		}
+
+		bool dump(const char* p, size_t sz)
+		{
+			memcpy(bufferPtr, p, sz);
+			bufferPtr += sz;
+
+			return true;
+		}
```
Comment (devshgraphicsprogramming) [inline include/irr/asset/CQuantNormalCache.h:396]: do the `load` and `dump` actually get called without polymorphism?

Comment (Przemog1) [reply include/irr/asset/CQuantNormalCache.h:396]: yup, check out parameters of `load` and `dump` functions
https://github.com/greg7mdp/parallel-hashmap/blob/master/parallel_hashmap/phmap_dump.h#L120

Comment (devshgraphicsprogramming) [reply include/irr/asset/CQuantNormalCache.h:396]: ah, sweet!

**Comment by devshgraphicsprogramming** [comment]
can I merge?

**Comment by Przemog1** [comment]
not yet, I've got few more things to do

**Comment by devshgraphicsprogramming** [comment]
mergable?

**Comment by Przemog1** [comment]
yes

**Pull request merged**